scripts/scripts.js

```diff
@@ -64,9 +64,11 @@ function buildHeroBlock(main) {
   const h2 = section.querySelector('h2');
   const picture = section.querySelector('picture');
   const cta = section.querySelector('a');
+  const list = section.querySelector('ul');
```

Comment:
the more I think about this, the more I think we should just grab every element in the section. so this whole but basically becomes:

```
  if (document.body.classList.contains('news')) {
    return;
  }

  const h1 = main.querySelector('h1');
  if (!h1) {
    return;
  }

  const section = h1.closest('div');
  const picture = section.querySelector('picture');
  if (picture) {
    const elems = [...section.childNodes];
    section.append(buildBlock('hero', { elems }));
    main.prepend(section);
  }
```

this accomplishes 2 things:

* no need to select individual elements, just whatever is in that section becomes part of the hero
* since we are taking all the elements, nothing gets left behind so we don't end up with an empty default content wrapper

This will likely affect the hero as a whole, so may require some js/css tweaks to it, but I think (and certainly open to discussion/feedback on this point) it makes the whole thing easier to manage across whatever variations they come up with,

Reply:
Getting all elements at the `scripts.js` level and sending them to the actual `block` impl makes more sense to me (from a dev standpoint)

I imagine it might make it confusing for authors adding other paragraphs in the hero section and refreshing the page w/o seeing any updates, and not knowing if it's a caching issue or another issue.